.golangci.yaml

```diff
@@ -2,6 +2,11 @@ run:
   timeout: 5m
 
 linters-settings:
+  gocritic:
+    enabled-tags:
+    - performance
+    disabled-checks:
```

Comment:
looks like enabling this finds an accidental comment: https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/stats_reporter.go#L86

Sounds like we just need to add a space after `//`? if so, I think this is good to enable

Reply:
Done.